applications/asset_tracker_v2/src/modules/cloud_module.c

```diff
@@ -945,7 +978,7 @@ static void on_sub_state_cloud_connected(struct cloud_msg_data *msg)
 		add_qos_message(msg->module.data.data.buffer.buf,
 				msg->module.data.data.buffer.len,
 				NEIGHBOR_CELLS,
-				QOS_FLAG_RELIABILITY_ACK_DISABLED,
+				QOS_FLAG_RELIABILITY_ACK_REQUIRED,
```

Comment:
This reasoning is not valid anymore because we don't wait for acknowledgement for neighbor cell measurement but there might be future implementations where we do wait for the response.